src/ts/helpers/heuristics.ts

```diff
@@ -69,6 +69,13 @@ export function isSecure(entry: WaterfallEntry) {
   return entry.name.indexOf("https://") === 0;
 }
 
+export function isPush(entry: WaterfallEntry) {
+  const harEntry = entry.rawResource;
+  // WebPageTest got 1 if an asset was pushed, Browsertime = true
+  console.log(harEntry["_was_pushed"]);
+  return ( harEntry["_was_pushed"] === true || Number(harEntry["_was_pushed"]) === 1) ? true : false;
```

Comment:
The conditional (?:) isn't needed here. I might even go with ```!!harEntry["_was_pushed"]```. Does WPT put a number (1) or a string ("1")? The Number conversion looks a bit strange.